types/node/fs.d.ts

```diff
@@ -2276,20 +2312,15 @@ declare module 'fs' {
             bytesRead: number;
             buffer: TBuffer;
         }>;
-    }
-    export interface ReadSyncOptions {
-        /**
-         * @default 0
-         */
-        offset?: number | undefined;
-        /**
-         * @default `length of buffer`
-         */
-        length?: number | undefined;
-        /**
-         * @default null
-         */
-        position?: ReadPosition | null | undefined;
+        function __promisify__<TBuffer extends NodeJS.ArrayBufferView>(
```

Comment:
The `__promisify__` variant functions I'm pretty sure should return a `Promise` and not have a `callback` parameter.